oxo.py

```diff
@@ -0,0 +1,133 @@
+class OxoBoard:
+    def __init__(self, x, y):
+
+        self.Board = {(0, 0): 0, (1, 0):0, (2, 0):0, (0, 1):0, (1, 1):0, (2, 1):0, (0, 2):0, (1, 2):0, (2, 2):0}
+
+    def get_square(self, x, y):
+
+        return self.Board[x,y]
+
+    def set_square(self, x, y, mark):
+
+        if self.Board[x, y] == 0:
+            self.Board[x, y] = mark
+            return True
+
+        else:
+            return False
+
+    def is_board_full(self):
+
+        SquareSearch = 0
+        for x in xrange(0, 3):
+
+            for y in xrange(0, 3):
+
+                if self.Board[x, y] > 0:
+                    SquareSearch += 1
+
+                if SquareSearch == 9:
+                    return True
+
+                elif [x, y] == (2, 2):

```

Comment:
This won't work -- a list is never equal to a tuple, even if they contain the same elements. I think you mean
```
[x, y] == [2, 2]
```
or
```
(x, y) == (2, 2)
```

However this is bad practice anyway -- if you want to execute something after the last iteration of a loop, just put it outside the loop.